libraries/AP_HAL/board/chibios.h

```diff
@@ -141,8 +141,15 @@
 #define HAL_USE_QUADSPI (HAL_USE_QUADSPI1 || HAL_USE_QUADSPI2)
 #define HAL_USE_OCTOSPI (HAL_USE_OCTOSPI1 || HAL_USE_OCTOSPI2)
 
-#if defined(STM32H7) || defined(STM32F7)
+#ifndef HAL_INS_RATE_LOOP
+#if defined(STM32H7) || defined(STM32F7) || (defined(STM32F4) && INS_MAX_INSTANCES == 1)
+/* F405 tested successfully with:
+   INS_GYRO_RATE = 1 (2kHz)
+   SCHED_LOOP_RATE = 200
+   FSTRATE_DIV = 2 (1kHz)
```

Comment:
```suggestion
   FSTRATE_ENABLE = 1 (Enabled-Dynamic)
   FSTRATE_DIV = 2 (1kHz)
```

I'm guessing here, could be one of the other options....

Reply:
Why was this conversation resolved with no action taken?

Reply:
FSTRATE_ENABLE is in there as you asked